docs/protocol/README.md

```diff
@@ -36,17 +36,25 @@ plus several extensions that are defined in the following documents:
 
 - [Binary tensor data extension](./extension_binary_data.md)
 - [Classification extension](./extension_classification.md)
-- [Model configuration extension](./extension_model_configuration.md)
-- [Model repository extension](./extension_model_repository.md)
 - [Schedule policy extension](./extension_schedule_policy.md)
 - [Sequence extension](./extension_sequence.md)
 - [Shared-memory extension](./extension_shared_memory.md)
+- [Model configuration extension](./extension_model_configuration.md)
```

Comment:
Is there a reason to move this lower? It looks like this list was alphabetic except for the last two: logging/parameters (which we should hopefully list alphabetically later too for consistency).

Reply:
It's moved so that the infer-related extensions are "grouped", given this parameter extension should be moved up. I think grouping by functionality is better than by alphabet. Alphabetic ordering works better when the reader knows what they are looking for.

Reply:
I see. Thanks for explaining, Guan.